package.json

```diff
@@ -22,7 +22,8 @@
     "bip38": "^2.0.2",
     "bs58": "^4.0.1",
     "classnames": "^2.2.6",
-    "eosjs": "^20.0.0",
+    "elliptic": "6.5.2",
+    "eosjs": "20.0.4-e2c667e.0",
```

Comment:
2 things here
 - We don't want to be merging any of this to master.  Merging to master is only what we do once we want to release the product (which we don't yet).  Please change the base of this branch to `develop`.
 - We will *never* reference an edge release of eosjs in a `master` PR.  I think this is just a consequence of point #1, but I wanted to acknowledge the issue separately just to ensure the training...

Reply:
Switched the PR to use develop instead of master.  Branch/commit were started from develop locally.